src/components/nve-label/nve-label.ts

```diff
@@ -56,28 +56,25 @@ export class NveLabel extends LitElement {
 
   private renderValueProperty() {
     if (this.value.length) {
+      // Vis value-property
+      // For å vise label i slot INNI tooltip-slot, må label-slot ha et navn
       return html`
       <label part="form-control-label" class="form-control__label" aria-hidden="false">
-        <slot name="label">${this.value}</slot>
+        <slot name="sillyRandomName">${this.value}</slot>
       </label>`
-    }
-    return html``; // value-property er ikke satt, så vi viser ikke denne delen
-  }
-
-  private renderSlottedContent() {
-    if (!this.value.length) {
+    } else {
+      // Vis evt. slot-innhold i stedet
       return html`
       <label part="form-control-label" class="form-control__label" aria-hidden="false">
-        <slot>${this.value}</slot>
+        <slot></slot>

```

Comment:
Shoelace bruker vel "trikset":
<slot name="label">{$this.value}</slot>

Da viser du label fra property i samme slot som du ev slotter inn. Hvis du da bruker slotted content i stedet vil et overskrive innholdet i slotten

Reply:
Vet ikke om jeg forstår deg riktig?
Jeg mener vi prøvde dette igår, men da fikk vi ikke vist slotted content. Prøvde nå også
```
  private renderValueProperty() {
    // Vis value-property
    // For å vise label i slot INNI tooltip-slot, må label-slot ha et navn
    return html`
      <label part="form-control-label" class="form-control__label" aria-hidden="false">
        <slot>${this.value}</slot>
      </label>`
  }
```
, men da vises ikke innhold i label når jeg gjør dette:
```
<nve-label>
            Ledetekst i <i>HTML</i>
          </nve-label>
```

Reply:
Implementasjonen du har nå viser value og ikke slotted dersom du angir begge deler.
Lurer på om problemet er i HasSlotController implementasjonen. Men det kan vi sikkert leve med. Lager issue på det